src/app/facebook-pixel-service/facebook-pixel-service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FacebookPixelService {
```

Comment:
How about `FacebookService` to reflect that we're loading the SDK, and the main tracking mechanism is JS rather than a pixel?